caikit/core/data_model/dataobject.py

```diff
@@ -88,9 +121,9 @@ def decorator(cls: Type) -> Type[DataBase]:
         # Make sure that the wrapped class does NOT inherit from DataBase
         error.value_check(
             "<COR95184230E>",
-            not issubclass(cls, DataBase),
-            "{} should not directly inherit from DataBase when using @schema",
```

Comment:
This was a fun learning experience for me. The `value_check` signature is strange and requires that `msg` be given as a kwarg while the `format` args come before the message. 🤷

Reply:
@gabe-l-hart yup! Most of our current usages of it are broken and don't create a message which is super fun.

If we're up for changing that I'd spin up a quick PR to make it match how logging works, like it was here before

Reply:
ohh! This is going to cause changes in a lot of value checks! Let me create a PR for this on the side

Reply:
Interesting! Yeah we saw this issue with BAM debugging too! Added a failed test here: https://github.com/caikit/caikit/pull/94. Can work on the fix as well